rust/otel-arrow-rust/.gitignore

```diff
@@ -11,3 +11,6 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+# These are per-developer
+memory-bank/
```

Comment:
Out of curiosity what is `memory-bank/`? Is this something related to your IDE?

By "These are per-developer", is the intention here that each dev, if they have some local files/directories in our local dev environment, that we should add them here?

Reply:
I have been using this prompt:
https://docs.cline.bot/prompting/cline-memory-bank
and it's all very new and I do not have any strong feelings about how a team could use this together.
The idea is that you can erase it and start over any time. It helps an agent remember what you are working on across time, but you have to hold it carefully I've found.